### ISSUE TITLE
German Translation

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#300]
```
Related topic:
http://andorstrail.com/viewtopic.php?f=6&t=329&p=13066#p13066
```

Original issue reported on code.google.com by `SamuelPl...@gmail.com` on 14 Mar 2012 at 1:01

Attachments:
- [German strings.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-300/comment-0/German strings.patch)

**Comment by GoogleCodeExporter** [comment]
```
Updated the patch.
```

Original comment by `SamuelPl...@gmail.com` on 17 Mar 2012 at 12:53

Attachments:
- [German Translation.zip](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-300/comment-1/German Translation.zip)

**Comment by GoogleCodeExporter** [comment]
```
Thanks, looks good. Committed in r243.
```

Original comment by `oskar.wi...@gmail.com` on 8 May 2012 at 8:20
- Changed state: **Ready**
- Added labels: **Type-Translation**, **Milestone-0.6.11**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.11. Closing issue.
```

Original comment by `oskar.wi...@gmail.com` on 9 Jun 2012 at 9:31
- Changed state: **Fixed**
